test/serializer/pure-functions/AbstractPropertyDelete.js

```diff
@@ -1,24 +1,40 @@
 // additional functions
 // abstract effects
 
-let obj1 = global.__abstract ? __abstract('object', '({get foo() { return "bar"; }})') : {get foo() { return "bar"; }};
-let obj2 = global.__abstract ? __abstract('object', '({foo:{bar:"baz"}})') : {foo:{bar:"baz"}};
+global.obj1 = { a: 1 };
+global.obj2 = { a: 2 };
+
+let obj1 = global.__abstract ? __abstract('object', 'obj1') : { a: 1 };
+let obj2 = global.__abstract ? __abstract('object', 'obj2') : { a: 2 };
+
```

Comment:
Why do we define both globals and regular variables with the same name? This seems a bit confusing to me.

Reply:
It's because the variable disappears from Prepack serialization if we don't.

Reply:
Let's change this to use `// add at runtime`